modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -33,14 +33,16 @@ spec:
         action: Accept  <1>
         cidr: 172.210.150.1/24 <2>
         protocol: SCTP
+        tcpFlags: SYN     <3>
```

Comment:
@Amoghrd added this and a link [here](https://github.com/openshift/openshift-docs/pull/80610/files#diff-2a7f18c8b04dd9d3c2c3958610e2279e9a65bedc901c51e511de310f46aac959R58) to the SYN flooding example, which should render right under this yaml.

Reply:
Now that I see this section https://80610--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/observing-network-traffic#network-observability-flowcollector-flowfilter-parameters_nw-observe-network-traffic ; I feel we can just add the tcpFlags parameter here instead of even adding to that YAML. Sorry I had missed this table previously

Reply:
Ah, yeah that's a good point tbh. No worries, this is what reviews are for :) I will add tcpFlags to this table and remove it from the YAML.

Reply:
Ok added here: https://github.com/openshift/openshift-docs/pull/80610/files#diff-c1f389cfa44881b2885e5709e478c820d4f98f140dbb208b231639b47ec4d159R41-R43

Reply:
Can remove from this YAML then

Reply:
this is not removed still

Reply:
Ah, sorry I think I looked at file and was paying attention to the 2nd collapsible example in the file, and saw there was no SYN there, and thought I'd removed it, but its actually the example above this one in the file 😵‍💫

Reply:
Ok cool this is showing as outdated now so I definitely got it 😅  my b @Amoghrd